Vagrantfile

```diff
@@ -9,31 +9,49 @@ if Gem::Version.new(Vagrant::VERSION) < Gem::Version.new("1.5.0")
 	exit 1
 end
 
+# Path to the root of the Chassis directory.
+chassis_dir = File.dirname(File.expand_path(__FILE__))
+
+# Load Chassis config from the Chassis root by default.
+chassis_config_dir = chassis_dir
+
+# Load Chassis config overrides from a custom directory, if configured.
+unless ENV['CHASSIS_CWD'].nil?
+	chassis_config_dir = ENV['CHASSIS_CWD']
+end
+
 # Check that submodules have been loaded
-if not File.exist?(File.join(File.dirname(__FILE__), "puppet", "modules", "apt", ".git"))
+if not File.exist?(File.join(chassis_dir, "puppet", "modules", "apt", ".git"))
 	puts "NOTICE: Submodules not found, updating for you"
 
-	if not system("git submodule update --init", :chdir => File.dirname(__FILE__))
+	if not system("git submodule update --init", :chdir => chassis_dir)
 		puts "WARNING: Submodules may be missing, and could not automatically\ndownload them for you."
 	end
 
 	# Extra new line, please!
 	puts
 end
 
+# Ensure we have a content directory to sync.
+chassis_content_dir = File.join(chassis_dir, "content")
+Dir.mkdir(chassis_content_dir) unless File.exists?(chassis_content_dir)
```

Comment:
@rmccue We need the `content` directory to exist before Vagrant can sync it to the virtual machine or it fails with the following error on the initial `vagrant up`:

```
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* The host path of the shared folder is missing: /absolute/path/to/Chassis/content
```